src/telliot_feeds/sources/price/spot/pulsechain_subgraph.py

```diff
@@ -69,7 +67,7 @@ async def get_price(self, asset: str, currency: str) -> OptionalDataPoint[float]
                 logger.warning("Timeout Error, No prices retrieved from Pulsechain Supgraph")
                 return None, None
 
-            except Exception as e:
+            except Exception:
```

Comment:
You want to include the unknown error in the logging message, so use `e` actually. Also elevate the logging level to ERROR